0.5/articles/webview.md

```diff
@@ -0,0 +1,325 @@
+
+If you need to support older versions of Android, try [Crosswalk](https://crosswalk-project.org). Crosswalk is a tool that brings the new Chromium webview to Android 4.0+. One downside is that the entire Chromium runtime gets bundled with your application. According to the [Crosswalk FAQ](https://crosswalk-project.org/#documentation/about/faq), this means 24KB web app can balloon into 19.63MB after it's packaged. The tradeoff is something to consider.
+
+## Recommended app structure
+
+The files that power your WebView should be entirely within the project's `src/main/assets` folder. Android reserves this directory for raw files that your app needs access to. Perfect for static web files!
+
+Inside of the `assets` folder, it's recommended to create a `www` folder to stash your web app. This folder is also where installed element dependences (e.g. `bower_components`) can go.
+
+### Using Bower to install elements
+
+**Tip** If you're not familiar with Bower, see
+[Installing elements](../docs/start/getting-the-code.html#installing-components).
+{: .alert .alert-success }
+
+Create a `bower.json` file in `src/main/assets/www` that lists your element dependencies. As an example, lets pull in all the paper and core elements:
+
+In `src/main/assets/www`, create **bower.json**:
+
+    {
+      "name": "PolymerWebView",
+      ...
+      "dependencies": {
+        "core-elements": "Polymer/core-elements#^{{site.latest_version}}",
+        "paper-elements": "Polymer/paper-elements#^{{site.latest_version}}"
+      }
+    }
+
+In the `www` directory, run `bower install`  to populate the `bower_components` folder.
+
+### Using HTML Imports
+
+Create `src/main/assets/www/elements.html`, an [HTML Import](../platform/html-imports.html) that includes all of the element imports your app will use:
+
+    <link rel="import" href="bower_components/core-drawer-panel/core-drawer-panel.html">
+    <link rel="import" href="bower_components/core-toolbar/core-toolbar.html">
+    <link rel="import" href="bower_components/core-icons/core-icons.html">
+    ...
+
+### Main page setup
+
+Create a main page, `src/main/assets/www/index.html` and use an HTML Import to load `elements.html`:
+
+    <!doctype html>
+    <html>
+    <head>
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <!-- Note: webcomponents.js is only needed in non-Chrome 36 browsers. It's included here for portability. -->
+      <script src="bower_components/webcomponentsjs/webcomponents.min.js"></script>
+      <link rel="import" href="elements.html">
+      ...
+    </head>
+    <body unresolved fullbleed>
+      ...
+    </body>
+    </html>
+
+It's worth noting that webcomponents.js isn't required for the Chromium 36+ Android WebView. However, if you're creating an app for multiple platforms, it's recommended to include the polyfills for portability.
```

Comment:
Should `webcomponents.js` be in backticks, or in regular text (as the name of the library)?